css/src/index.css

```diff
@@ -5,10 +5,55 @@
 ** @import "citizennet-css"
 */
 
-
 /* Import utilities after our CSS to avoid overwriting them. */
 @tailwind utilities;
 
-/* CitizenNet utilities
-** @import "citizennet-css"
-*/
+/* cn utilities */
+
+.cc-teal { caret-color: config('colors.teal'); }
+.no-content { content: ''; }
+.no-outline { outline: none; }
+.scale-0 { transform: scale(0); }
+.scale-1 { transform: scale(1); }
+
+/* pseudo utilities */
+
+.after\:absolute:after { @apply .absolute; }
+.after\:bg-teal:after { @apply .bg-teal; }
```

Comment:
Is it necessary to specialize these? Can we have this apply to any color, for example, like `hover:`? It could get messy adding these as colors change.

Reply:
tailwinds doesn't yet provide classes for several pseudo selectors like :before and :after. As far as I can tell, we have to specify these ourselves.